workspaces/Cargo.toml

```diff
@@ -18,6 +18,7 @@ bs58 = "0.4"
 cargo_metadata = { version = "0.14.2", optional = true }
 chrono = "0.4.19"
 dirs = "3.0.2"
+futures = "0.3"
```

Comment:
originally used futures a bit more than just importing `BoxFuture`, but seems like that's the only import, I directly pulled it into `crate::rpc::BoxFuture` so that we don't need this import